src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -451,7 +451,9 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         auto settings{ _core.Settings() };
         auto bgColor = til::color{ _core.FocusedAppearance().DefaultBackground() };
-        if (settings.UseAcrylic())
+        // GH#11743: Make sure to use the Core's current UseAcrylic value, not
+        // the one from the settings.

```

Comment:
nit: It's clear from the code that it's using the core and not the settings UseAcrylic value. This comment can be improved by summarizing why this is important.